spec/rubocop/cop/rails/has_many_or_has_one_dependent_spec.rb

```diff
@@ -79,5 +91,28 @@ class Person
         RUBY
       end
     end
+
+    context 'with_options dependent: :destroy' do
+      it 'register an offense for not `with_options dependent: :destroy`' do
```

Comment:
registers an offense when `dependent: :destroy` is not present (or something like this)